docs/getting-started.md

```diff
@@ -0,0 +1,18 @@
+# Getting Started
+
+Consult the full initial setup instructions on the [HPCCloud-deploy repository](https://github.com/Kitware/HPCCloud-deploy) and then come back here.
```

Comment:
I think we want to have step by step instructions here, even if it means some duplication:

1. Install prerequisites
2. git clone https://github.com/Kitware/HPCCloud-deploy
3. cd HPCCloud-deploy
4. DEMO=1 vagrant up